baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -41,6 +41,7 @@ The definition of `OWS` is taken from [[RFC7230]], Section 3.2.3: https://tools.
 ### list
 List of key-value pairs with optional properties attached.
 It can not be guaranteed that keys are unique.
+The list order SHOULD be preserved when mutating the list.
```

Comment:
How did this come up? It seems pretty onerous to implement.

Reply:
@SergeyKanzhelev mentioned that in the end, we came up with a SHOULD requirement here. @yurishkuro are you objecting against making the list order a soft requirement at all?

Reply:
I believe the SHOULD is what many were advocating for to be able to deal with dups successfully. At least for proxies it will be best to preserve the order. I agree that many mutation implementations will be implementing it as a map, so it will be hard to preserve the order. Maybe we can say SHOULD preserve oder of duplicated entries

Reply:
Changed the wording to "The order duplicate entries SHOULD be preserved when mutating the list."